hrjobcontract/js/test/controllers/modal/modal-contract-new.controller.spec.js

```diff
@@ -69,6 +69,10 @@ define([
         Individual: 'Individual'
       };
 
+      it('sets the contract property', function () {
```

Comment:
it's not descriptive enough. sets to what?

Reply:
> @deb1990 add the first bad commit in the PR description, and also reference what file we are talking about, cause you're not mentioning it

Both has been Fixed.